lib/node/fullnode.js

```diff
@@ -43,6 +43,9 @@ class FullNode extends Node {
     // SPV flag.
     this.spv = false;
 
+    // Prune flag
+    this.prune = this.config.bool('prune');
```

Comment:
nit: `this.config.bool('prune')` is used three places for `FullNode` for `Chain`, `TXIndexer`, `AddrIndexer` and could use `this.prune` instead, and the value could default to `false` with `this.config.bool('prune', false)`, it's current value is `null`. The abstract node at `lib/node/node.js` should probably have `this.prune = false`, similar to the `this.spv = false`.

Reply:
Yes. Great idea, adding.